build.gradle.kts

```
@@ -215,6 +215,11 @@ allprojects {
             dependsOn(publish)
         }
 
+        // Disable publishing a bunch of unnecessary Gradle metadata files
```

```suggestion
        // Don't publish Gradle metadata for now until verifying they work well.
```

Technically they're not unnecessary for Gradle :) But it's true that since we need the POMs to work well anyways, they don't provide much value at least for now.

---

Fair enough! I was just going off other AWS projects published to maven central and saw they didn't include the metadata. Will update comment.